conf/robottelo-manifest-content.conf

```diff
@@ -0,0 +1,7 @@
+# This file contains subscription names (one per line) expected in robottelo-automation manifest.
+# Subscriptions named here must be in the manifest and no other subs are allowed.
+Red Hat Beta Access;8a85f99a710f3ae90171417b9a7d7907;75
```

Comment:
i'm not sure it is okay to share the pool ids?

Reply:
I wasn't sure what this file was when I pushed this commit. Should I remove this from my PR?

Reply:
I think we should remove the pool id from here, @lpramuk any thought?

Reply:
@Griffin-Sullivan You should be careful while pushing something public, and we shouldn't put any internal URLs/details on GH, pls remove Jira URL from the description.

Reply:
Ok I removed it. Again, I didn't know what that was, and had no intention of including it in this commit. I would love if someone could provide more details on what the file is, and how it could have been included in this PR.